swift/extractor/invocation/CodeQLDiagnosticsConsumer.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <swift/AST/DiagnosticConsumer.h>
+#include "swift/extractor/infra/SwiftLocationExtractor.h"
+
+namespace codeql {
+
+class TrapDomain;
+
+class CodeQLDiagnosticsConsumer : public swift::DiagnosticConsumer {
```

Comment:
nit: shouldn't this be `SwiftDiagnosticsConsumer` to be consistent with other names?